fastify.js

```diff
@@ -611,6 +612,22 @@ function fastify (options) {
       rejectReady = reject
     })
 
+    if (typeof signal?.addEventListener === 'function') {
```

Comment:
We are supporting two kinds of `signal` signatures, `on` and `addEventListener`. Let's also add support for it